examples/simple-trino-cluster.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: trino.stackable.tech/v1alpha1
+kind: TrinoCluster
+metadata:
+  name: simple
+spec:
+  version: "0.0.362"
+  nodeEnvironment: production
+  hiveReference:
+    namespace: default
+    name: simple-derby
+  opa:
+    namespace: default
+    name: simple
+  s3Connection:
+    endPoint: changeme
+    accessKey: changeme
+    secretKey: changeme
+    sslEnabled: false
+    pathStyleAccess: true
+  coordinators:
+    roleGroups:
+      default:
+        selector:
+          matchExpressions:
```

Comment:
Is there a reason why we bind to explicit hosts here instead of using kubernetes.io/os ?